docs/scripts/mdx/builder.ts

```diff
@@ -0,0 +1,38 @@
+import { writeFile } from "fs/promises"
+import path from "path"
+import { glob } from "glob"
+import { generateMDX } from "./document"
+import { initMDX } from "./package"
+import type { DocumentMap } from "./utils"
+import { getLocale, getSlug, MDX_PATH } from "./utils"
+
+async function builderMDX() {
+  initMDX()
+
+  const filePaths = await glob("contents/**/*.mdx")
+  const count = filePaths.length
+
+  const map: DocumentMap = {}
+  const paths: string[] = []
+
+  await Promise.all(
+    filePaths.map(async (filePath) => {
+      const locale = getLocale(filePath)
+      const slug = getSlug(filePath)
+
+      const mdx = await generateMDX(filePath)
+
```

Comment:
We should add `.replace(/\\/g, "/")` to this to fix issue on Windows.

```suggestion

      const mdx = await generateMDX(filePath.replace(/\\/g, "/"))

```

Reply:
Fixed